client/i18n/locales/english/intro.json

```
@@ -2371,7 +2371,7 @@
       "lecture-understanding-how-to-work-with-floats-and-positioning-in-css": {
         "title": "Understanding How to Work with Floats and Positioning in CSS",
         "intro": [
-          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn about position properties like <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code>. YOu will also learn about the <code>z-index</code> property."
+          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn how to use <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code> positioning. You will also learn about the <code>z-index</code> property."
```

```suggestion
          "In these lecture videos, you will learn how to use CSS positioning and floats. You will learn about <code>absolute</code>, <code>relative</code>, <code>fixed</code>, and <code>sticky</code> positioning. You will also use the <code>z-index</code> property."
```
Too much repetition made it feel clumsy.